mysql-test/suite/versioning/t/replace.test

```diff
@@ -23,10 +23,11 @@ select *, current_row(row_end) as current, count(row_start) = 1 as unique_start,
 --echo # MDEV-15990: REPLACE on a precise-versioned table
 --echo # returns duplicate key error (ER_DUP_ENTRY)
 replace t values (1, 3), (1, 30), (1, 40);
-select *, current_row(row_end) as current, count(row_start) = 1 as unique_start,
-    row_start != 0 as start_nonzero
-  from t for system_time all
-  group by row_start order by x, row_end;
+select *, current_row(row_end) as current, row_start != 0 as start_nonzero
+  from t for system_time all order by x, row_end;
+
+select count(row_start) != 1 as empty from t for system_time all
+  group by row_start having empty = 1;
```

Comment:
This is hard to decipher. Why do you need such complex query?

Reply:
Well we can just add `unique` constraint to `row start`, but it will break by making a slightly different sequence of `replace`s. But the query will survive